datasets/dataset_setup.py

```diff
@@ -55,7 +55,7 @@
 Example command:
 
 python3 datasets/dataset_setup.py \
-  --data_dir=~/data \
+  --data_dir=~/data/imagenet \
```

Comment:
undo?

Reply:
No this is intentional. Unless we want to automate naming the directories for a specific dataset?